t/043-shdict.t

```diff
@@ -2471,3 +2471,90 @@ error
 lua_shared_dict "dogs" is already defined as "dogs"
 --- error_log
 [emerg]
+
+
+
+=== TEST 94: incr expire test new key
+--- http_config
+    lua_shared_dict dogs 1m;
+--- config
+    location = /test {
+        content_by_lua '
```

Comment:
I'll do a future PR changing all the shdict tests to _block, not just these ones. It's an easy regex.